ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -276,7 +276,21 @@ public void init() {
           String id = req.params(":id");
           return util.getJson(validator.getFullValidation(util.getMetacardById(id)));
         });
+    get(
```

Comment:
✏️  I added these as new lines just in case we need the validation endpoint without the storeId. Not sure if it's better to collapse them.